lib/arduino_ci/cpp_library.rb

```diff
@@ -37,11 +37,14 @@ class CppLibrary
 
     # @param base_dir [Pathname] The path to the library being tested
     # @param arduino_lib_dir [Pathname] The path to the libraries directory
-    def initialize(base_dir, arduino_lib_dir)
+    def initialize(base_dir, arduino_lib_dir, exclude_dirs = [])
```

Comment:
`exclude_dirs` should not be an optional argument here.  You'll need update all instances of `CppLibrary.new` (i.e. the spec files) to explicitly pass in the empty array.